src/main/java/uk/gov/hmcts/reform/authorisation/validators/AuthTokenValidator.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.authorisation.validators;
+
+import java.util.List;
+
+interface AuthTokenValidator {
```

Comment:
I like the idea, but since there is only one implementation maybe we don't need the interface just yet?

Reply:
There was a suggestion to add cached token validator. But tempted to remove as, for some reason, checkstyle fails on interfaces and `List<>` types. Haven't tested the theory with "on interfaces" yet